### ISSUE TITLE
calcite-block does not read properly in NVDA when setting focus

### ISSUE THREAD

**Issue by jordantsanz** [Esri/calcite-design-system#10713]
### Check existing issues

- [X] I have [checked for existing issues](https://github.com/Esri/calcite-design-system/issues) to avoid duplicates and reviewed the [Accessibility page](https://developers.arcgis.com/calcite-design-system/foundations/accessibility) for guidance.

### Summary

When I click a calcite-button and a calcite-block is appended to the document with a readonly calcite-combobox inside of it, and focus is immediately set onto the calcite-block. 
When using VoiceOver, this reads out "Collapse New filter, expanded, button, group, article". 
However, when using NVDA, it tries to say "Collapse button expanded New Filter" but also says "Group 1 of 1" at the same time, and that is all the user hears.
I assume the "Group 1 of 1" is coming from the combobox within the block.

### Actual Behavior

1. Launch codepen and turn on NVDA
2. Press enter on the button that adds a new calcite-block to the document
3. Focus is set onto the block, and "Group 1of1" is read out. This seems to be the block's inner content, as there is a readonly combobox that has "Group" written inside of it.

### Expected Behavior

1. Launch codepen and turn on NVDA
2. Press enter on the button that adds a new calcite-block to the document
3. Focus is set onto the block, and "Collapse button expanded New Filter Group 1of1" is read out as this is the title of the block and its relevant attributes (expanded, that it is a collapse button) and then the content that has appeared inside of the block

### Reproduction Sample

https://codepen.io/jordansanz/pen/YzmJgYp?editors=100

### Reproduction Steps

1. Launch codepen and turn on NVDA
2. Press enter on the button that adds a new calcite-block to the document
3. Focus is set onto the block, and "Group 1of1" is read out

### Reproduction Version

2.13.2

### Relevant Info

Windows NVDA needed

### Priority impact

impact - p1 - need for current milestone

### Calcite package

- [X] @esri/calcite-components
- [ ] @esri/calcite-components-react
- [ ] @esri/calcite-design-tokens
- [ ] @esri/eslint-plugin-calcite-components

### Esri team

ArcGIS Hub

**Comment by geospatialem** [comment]
In testing it appears this is a result of lack of support with the [`aria-controls` attribute](https://a11ysupport.io/tech/aria/aria-controls_attribute) in providing context through assistive technologies and browsers - as the context is provided with JAWS, VoiceOver, and NVDA with Firefox -- but not NVDA in Chromium browsers.

The `aria-controls` attribute is recommended, however user-facing support is, unfortunately, poor. So this seems to be a result of NVDA browser support

> Important: The aria-controls attribute has expectations that are not directly testable by end users. Continue to use it if it is required by the specification, even if user-facing expectation support is poor. For more information, see [FAQ: What about expectations that are not directly testable by users?](https://a11ysupport.io/faq#what-about-expectations-that-are-not-directly-testable-by-users%3F).

Closing the above as unplanned as a result.

cc @FelFly
